extension/lib/tracker.js

```diff
@@ -106,6 +106,10 @@ class Tracker {
         return $("#" + this.readableDomIds[containerId]);
     }
 
+    getFirstContainer() { // Added to access container ID
```

Comment:
was this file reverted?
i think we agreed that
- getFirstContainer() is not used
- window.display below should not be used cos we want classes like tracker to be self contained.

Reply:
Yep; see above comments (this branched from the most recent pull request pre-review, when these problems were still in there). I was hoping they'd be fixed in merge conflicts, it appears this did not, will fix